### PR TITLE
Dashboard Updates

### PR DESCRIPTION
**Pull request by mfixstsci**
Addressing #783 and #756

### CODE REVIEW

File: jwql/website/apps/jwql/bokeh_dashboard.py [hunk 1]
```diff
@@ -44,6 +44,21 @@
...
+    Parameters
+    ----------
+    figure: bokeh figure object
+
+    Returns
+    -------
+    None
+    """
+    yaxis = figure.select(dict(type=Axis, layout="left"))[0]
+    yaxis.formatter.use_scientific = False
```
Comment (bourque) [inline jwql/website/apps/jwql/bokeh_dashboard.py:59]: Glad that this was relatively straightforward!

**Review by bourque — COMMENTED** [review]
@mfixstsci I thought of one more thing that I think would make sense to add to this PR -- and that is to add the handful of `jwql_monitors` (e.g. `generate_preview_images`, `monitor_filesystem`, etc.) to the table in the dashboard, so that our team can keep track of those too.

Is that something you could add here?  Or if you don't have the time right now we could make a separate issue for it.  It doesn't really matter on my end.  

As far as the code changes you currently have here, they look great!

File: jwql/website/apps/jwql/bokeh_dashboard.py (hunk 1 above)
Comment (mfixstsci) [reply jwql/website/apps/jwql/bokeh_dashboard.py:59]: Yeah, me too. Figured we might want to use it in other parts of the package so I made it a function.

**Comment by mfixstsci** [comment]
> @mfixstsci I thought of one more thing that I think would make sense to add to this PR -- and that is to add the handful of `jwql_monitors` (e.g. `generate_preview_images`, `monitor_filesystem`, etc.) to the table in the dashboard, so that our team can keep track of those too.
> 
> Is that something you could add here? Or if you don't have the time right now we could make a separate issue for it. It doesn't really matter on my end.
> 
> As far as the code changes you currently have here, they look great!

This should totally be possible assuming they output logs! I will just have to add a call to `update_monitor_table`. I will look into it this afternoon.

**Comment by mfixstsci** [comment]
All of the monitors are failing, but conceptually, it works just fine. @bourque 
Screen Shot 2021-09-02 at 10 24 45 AM

**Comment by bhilbert4** [comment]
I bet none of the logs for those monitors have been set up to contain the success line at the end...

**Comment by mfixstsci** [comment]
> I bet none of the logs for those monitors have been set up to contain the success line at the end...

I just checked them out @bhilbert4:

```
monitor_mast: 09/01/2021 02:01:20 AM INFO: Completed Successfully
monitor_filesystem: 09/01/2021 01:54:32 AM INFO: Completed Successfully
monitor_cronjobs: 09/01/2021 02:30:13 AM INFO: Completed Successfully
generate_preview_images: 09/02/2021 10:23:14 AM CRITICAL: CRASHED
generate_proposal_thumbnails: 09/01/2021 03:00:17 AM INFO: Completed Successfully
```

The failure only has a single log, one that I created during testing.

**Comment by mfixstsci** [comment]
I just opened an older log:
`generate_preview_images: 07/22/2021 21:42:50 PM INFO: Completed Successfully`

LGTM 🤷‍♂️

**Comment by mfixstsci** [comment]
@bhilbert4 I am assuming its because I am running the monitors locally. I am getting issues with a local path which is causing the crashes.

**Comment by mfixstsci** [comment]
Changes were made @bourque should be ready to go!

**Review by bourque — APPROVED** [review]
Looks awesome, thanks @mfixstsci!

**Pull request merged**